src/wallet/wallet.cpp

```diff
@@ -1650,6 +1699,59 @@ void CWallet::AvailableCoins(vector<COutput>& vCoins, bool fOnlyConfirmed, const
 
     {
         LOCK2(cs_main, cs_wallet);
+#if 1
+        set<const CWalletTx*> rejectedCoins;
+        for (UnspentsMap::const_iterator uit = mapTxUnspents.begin(); uit != mapTxUnspents.end(); ++uit)
+        {
+            const COutPoint &outpoint = uit->first;
+            const uint256& wtxid = outpoint.hash;
+            const CWalletTx* pcoin = (uit->second).first;
+
+            if(rejectedCoins.find(pcoin) != rejectedCoins.end())
+                continue;
+
+            if (!CheckFinalTx(*pcoin))
+            {
+                rejectedCoins.insert(pcoin);
+                continue;
+            }
+
+            if (fOnlyConfirmed && !pcoin->IsTrusted())
+            {
+                rejectedCoins.insert(pcoin);
+                continue;
+            }
+
+            if (pcoin->IsCoinBase() && pcoin->GetBlocksToMaturity() > 0)
+            {
+                rejectedCoins.insert(pcoin);
+                continue;
+            }
+
+            int nDepth = pcoin->GetDepthInMainChain();
+            if (nDepth < 0)
+            {
+                rejectedCoins.insert(pcoin);
+                continue;
+            }
+
+            if(!(pcoin->vout[outpoint.n].nValue > 0 || fIncludeZeroValue))
+                continue;
+
+            if(IsLockedCoin(wtxid, outpoint.n))
+                continue;
+
+            if(coinControl && coinControl->HasSelected() && !coinControl->fAllowOtherInputs && !coinControl->IsSelected(wtxid, outpoint.n))
+                continue;
+
+            isminetype mine = (uit->second).second;
+            vCoins.push_back(COutput(pcoin, outpoint.n, nDepth,
+                                     ((mine & ISMINE_SPENDABLE) != ISMINE_NO) ||
+                                     (coinControl && coinControl->fAllowWatchOnly && (mine & ISMINE_WATCH_SOLVABLE) != ISMINE_NO)));
+        }
+
+        LogPrintf("Unspent count: %d %d\n", vCoins.size(), mapTxUnspents.size());
```

Comment:
IMO this can be removed, because was used only to debugging the execution in DEV right?